### ISSUE TITLE
Command failed: no G executing on thread 0

### ISSUE THREAD

**Issue by npcode** [derekparker/delve#468]
1. What version of Delve are you using (`dlv version`)?
   
   Delve Debugger
   Version: 0.11.0-alpha
   Build: 
2. What version of Go are you using? (`go version`)?
   
   go version go1.6 linux/amd64
3. What operating system and processor architecture are you using?
   
   OS: Linux 4.4.5-1-ARCH
   processor arch: x86_64
4. What did you do?
   1. Run `dlv attach 1` to debug a running [docker](https://github.com/docker/docker) daemon process. 
   2. Set a breakpoint
   3. Run 'n' command to step over next source line.
5. What did you expect to see?
   
   Step over to next source line.
6. What did you see instead?
   
   "Command failed: no G executing on thread 0" error as follows:
   
   ```
   > github.com/docker/libnetwork.(*network).CreateEndpoint() ./vendor/src/github.com/docker/libnetwork/network.go:740 (hits total:3) (PC: 0x8a6e60)
   735:                 if mac, ok := opt.(net.HardwareAddr); ok {
   736:                         ep.iface.mac = mac
   737:                 }
   738:         }
   739:
   => 740:         ipam, err := n.getController().getIPAM(n.ipamType)
   741:         if err != nil {
   742:                 return nil, err
   743:         }
   744:
   745:         if ipam.capability.RequiresMACAddress {
   (dlv) n
   Command failed: no G executing on thread 0
   ```

**Comment by aarzilli** [comment]
Apparently in the docker binary the G offset is completely different

```
	docker.go:17	0x40ca00	64488b0c25c8ffffff		mov rcx, qword ptr fs:[0xffffffc8]
```

just compiling something with cgo still produces the good old:

```
	cgotest.go:12	0x4014c0	64488b0c25f8ffffff		mov rcx, qword ptr fs:[0xfffffff8]
```

Hard to say what causes this since docker build process involves downloading a whole operating system.

The `thread 0` thing is just cosmetic and I will push a fix for that.

**Comment by joegrasse** [comment]
@aarzilli just curious, how did you produce the output to show the G offset in https://github.com/derekparker/delve/issues/468#issuecomment-200294856?

**Comment by aarzilli** [comment]
`disasm -l main.main`, but it's the first instruction of almost any function.

**Comment by heschik** [comment]
This should be fixed by #883 if someone wants to test and close it.
